airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/paginators/default_paginator.py

```diff
@@ -160,3 +160,68 @@ def _get_request_options(self, option_type: RequestOptionType) -> Mapping[str, A
             if option_type != RequestOptionType.path:
                 options[self.page_size_option.field_name] = self.pagination_strategy.get_page_size()
         return options
+
+
+class PaginatorTestReadDecorator(Paginator):
```

Comment:
Is there a reason why Paginator does not inherit ABC? As of now, we won't get a test error if a new method is added to Paginator